pkg/migration/controllers/clusterpair.go

```diff
@@ -220,3 +230,45 @@ func (c *ClusterPairController) createCRD() error {
 
 	return apiextensions.Instance().ValidateCRD(resource, validateCRDTimeout, validateCRDInterval)
 }
+
+func createBackupLocationOnRemote(remoteConfig *restclient.Config, clusterPair *stork_api.ClusterPair) error {
+	if bkpl, ok := clusterPair.Spec.Options[stork_api.BackupLocationResourceName]; ok {
+		remoteClient, err := storkops.NewForConfig(remoteConfig)
+		if err != nil {
+			return err
+		}
+		client, err := kubernetes.NewForConfig(remoteConfig)
+		if err != nil {
+			return err
+		}
+		ns, err := core.Instance().GetNamespace(clusterPair.Namespace)
+		if err != nil {
+			return err
+		}
+		// Don't create if the namespace already exists on the remote cluster
+		_, err = client.CoreV1().Namespaces().Get(clusterPair.Namespace, metav1.GetOptions{})
+		if err != nil {
+			// create namespace on destination cluster
+			_, err = client.CoreV1().Namespaces().Create(&v1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:        ns.Name,
+					Labels:      ns.Labels,
+					Annotations: ns.Annotations,
+				},
+			})
+			if err != nil && !errors.IsAlreadyExists(err) {
+				return err
+			}
+		}
+		// create backuplocation on destination cluster
+		resp, err := storkops.Instance().GetBackupLocation(bkpl, clusterPair.GetNamespace())
+		if err != nil {
+			return err
+		}
+		resp.ResourceVersion = ""
```

Comment:
Don't you also have to remove the UUID, self link and other members from the metadata?

Reply:
only  setting `resourceVersion` restrict resource creation on remote cluster